payment_method_object.go

```diff
@@ -0,0 +1,51 @@
+package easypost
+
+import (
+	"errors"
+	"strings"
+)
+
+type PaymentMethodObject struct {
```

Comment:
Is there a reason this is called `PaymentMethodObject` instead of just `PaymentMethod`? It's inherently an object and doesn't need that in the name nor do we use that elsewhere to my knowledge. Shipment is just Shipment.

Reply:
Per our internal conversations, `PaymentMethod` is the name of the summary object that wraps the `primaryPaymentMethod` and `secondaryPaymentMethod` object. `PaymentMethodObject` represents a credit card or a bank account.